lib/build-image-pipeline.ts

```diff
@@ -29,6 +29,10 @@ export interface BuildImagePipelineProps extends cdk.StackProps {
   readonly dataBucket: s3.IBucket;
   /** The ECR Repository to push to. */
   readonly repository: IRepository;
+  /** Access logging bucket to use */
+  accessLoggingBucket?: s3.Bucket;
```

Comment:
We should also expose the prefix for sharing a bucket: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3.Bucket.html#serveraccesslogsprefix

Reply:
agree

Reply:
done